"cancel CsvStream during iteration does not leak file" test is flaky

```

[encoding/csv/stream] cancel CsvStream during iteration does not leak file => ./encoding/csv/stream_test.ts:311:6
error: AssertionError: Test case is leaking async ops.

 - 1 async operation to op_read was started in this test, but never completed.

To get more details where ops were leaked, run again with --trace-ops flag.
    at assert (deno:ext/web/00_infra.js:295:13)
    at asyncOpSanitizer (deno:cli/js/40_testing.js:233:13)
    at async resourceSanitizer (deno:cli/js/40_testing.js:378:7)
    at async exitSanitizer (deno:cli/js/40_testing.js:435:9)
    at async Object.applyPermissions [as fn] (deno:cli/js/40_testing.js:537:9)
    at async runTest (deno:cli/js/40_testing.js:840:7)
    at async runTests (deno:cli/js/40_testing.js:1098:22)

```

- https://github.com/denoland/deno_std/actions/runs/4074683204/jobs/7020120482
- https://github.com/denoland/deno_std/actions/runs/4070151225/jobs/7010987705

--- a/csv/csv_parse_stream_test.ts
+++ b/csv/csv_parse_stream_test.ts
@@ -12,6 +12,7 @@ import {
 import type { AssertTrue, IsExact } from "../testing/types.ts";
 import { fromFileUrl, join } from "../path/mod.ts";
 import { StringReader } from "../io/string_reader.ts";
+import { delay } from "../async/delay.ts";
 
 const testdataDir = join(fromFileUrl(import.meta.url), "../testdata");
 const encoder = new TextEncoder();
@@ -373,9 +374,6 @@ Deno.test({
   name:
     "[csv/csv_parse_stream] cancel CsvParseStream during iteration does not leak file",
   permissions: { read: [testdataDir] },
-  // TODO(kt3k): Enable this test on windows.
-  // See https://github.com/denoland/deno_std/issues/3160
-  ignore: Deno.build.os === "windows",
   fn: async () => {
     const file = await Deno.open(join(testdataDir, "large.csv"));
     const readable = file.readable.pipeThrough(MyTextDecoderStream())
@@ -383,6 +381,8 @@ Deno.test({
     for await (const _record of readable) {
       break;
     }
+    // FIXME(kt3k): Remove this delay.
+    await delay(100);
   },
 });
 

